### PR TITLE
Clarify SameSite restrictions

### PR DESCRIPTION
**Pull request by mreichhoff**
- [x] At least two implementers are interested (and none opposed):
   * N/A; editorial only
- [x] [Tests](https://github.com/web-platform-tests/wpt) are written and can be reviewed and commented upon at:
   * N/A; editorial only
- [x] [Implementation bugs](https://github.com/whatwg/meta/blob/main/MAINTAINERS.md#handling-pull-requests) are filed:
   * N/A; editorial only

### CODE REVIEW

**Comment by mreichhoff** [comment]
sending to @johannhof; this is based on a brief conversation yesterday.

Thanks!

File: tag-security-questionnaire.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 The Storage Access API enables the removal of cross-site cookies. Specifically, it allows the authenticated embeds use case to continue to work. As such, the API provides a way for developers to re-gain access to cross-site cookies, albeit under further constraints.
 
-A nested Document gains access to the same cookies it has as the active document of a top-level browsing context when it calls `document.requestStorageAccess()` and is returned a resolving Promise. With these cookies it can authenticate itself to the server and load user-specific information.
+A nested Document gains access to the same cookies it has as the active document of a top-level browsing context when it calls `document.requestStorageAccess()` and is returned a resolving Promise (while this doesn't change the threat model, `SameSite` restrictions still apply; in practice, only `SameSite=None` cookies should be granted).
```
Comment (johannhof) [inline tag-security-questionnaire.md:6]: Hmm I'm not a fan of "in practice" here, slightly reworded suggestion:

`SameSite restrictions will still apply, i.e. SameSite=Lax or Strict cookies will not be available, which is important for security but doesn't affect the privacy considerations)`

This sentence most notably doesn't care what the default state in a particular browser is.

Thoughts? :)

Comment (mreichhoff) [reply tag-security-questionnaire.md]: updated it with this wording basically verbatim

**Review by johannhof — APPROVED** [review]
Thanks!

**Pull request merged**